core_lib/src/interface/scribblearea.cpp

```diff
@@ -902,8 +902,8 @@ void ScribbleArea::handleDrawingOnEmptyFrame()
                 KeyFrame* dupKey = previousKeyFrame->clone();
                 layer->addKeyFrame(frameNumber, dupKey);
                 mEditor->scrubTo(frameNumber);
-                break;
             }
+            break;
```

Comment:
No no no, this was right before. We intentionally want to create a new key in the duplicate mode if no previous key exists so it's as if it is duplicating the empty frames. Hence why there there are comments directly below this explaining that we want to fall through if the previous keyframe does not exist. @chchwy Please revert this.

Reply:
Sorry, my bad :(

Reply:
fixed.

Reply:
If we move the comments to right below the close bracket on line 909, it apparently will remove the warning on GCC. See https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wimplicit-fallthrough. Wouldn't be a bad idea to do this.

Reply:
Sorry, my reckless